apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -77,6 +77,9 @@ function MyTokens() {
         () =>
             data?.reduce(
                 (acc, coinBalance) => {
+                    acc.recognized.push(coinBalance);
+                    acc.pinned.push(coinBalance);
+                    acc.unrecognized.push(coinBalance);
                     if (
```

Comment:
do we need to remove the below logic? looks like everything is doubling up 

<img width="405" alt="image" src="https://github.com/MystenLabs/sui/assets/122397493/d8ae3bee-f858-4125-81f4-59e49159fb72">

Reply:
Ooops this was my testing logic lol